src/Microsoft.AspNetCore.SignalR.Redis/RedisDependencyInjectionExtensions.cs

```diff
@@ -8,13 +8,27 @@
 
 namespace Microsoft.Extensions.DependencyInjection
 {
+    /// <summary>
+    /// Extension methods for setting up <see cref="RedisHubLifetimeManager{THub}"/> in an <see cref="ISignalRServerBuilder" />.
```

Comment:
HubLifetimeManager isn't really a user-visible concept, they're looking for scale-out when they use this API. And the term "scale-out" is familiar to SignalR users. How about something like:

"Extension methods for configuring Redis-based scale-out for a SignalR Server"